lib/generators/react_on_rails/react_generator.rb

```diff
@@ -0,0 +1,132 @@
+        data << "// It is important that generated/vendor-bundle must be before "
+        data << "bootstrap since it is exposing jQuery and jQuery-ujs\n"
+        data << "//= require generated/vendor-bundle\n"
+        data << "//= require generated/app-bundle\n"
+        data << "//= require react_on_rails\n"
+        data << "\n"
+
+        application_js_path = "app/assets/javascripts/application.js"
+        application_js = dest_file_exists?(application_js_path) || dest_file_exists?(application_js_path + ".coffee")
+        if application_js
+          prepend_to_file(application_js, data)
+        else
+          puts_setup_file_error("#{application_js} or #{application_js}.coffee", data)
+        end
+      end
+
+      def create_react_directories
+        dirs = %w(actions components constants middlewares startup utils)
+        dirs.each { |name| empty_directory_with_keep_file "client/app/#{name}" }
+      end
+
+      def copy_base_files
+        template "react_on_rails.rb", "config/initializers/react_on_rails.rb"
+        copy_file "lib/tasks/assets.rake", "lib/tasks/assets.rake"
+        copy_file "client/README.md", "client/README.md"
+        copy_file "client/server.js", "client/server.js"
+        copy_file "client/.babelrc", "client/.babelrc"
+        copy_file "client/webpack.client.base.config.js", "client/webpack.client.base.config.js"
+        copy_file "client/webpack.client.hot.config.js", "client/webpack.client.hot.config.js"
+        copy_file "client/webpack.client.rails.config.js", "client/webpack.client.rails.config.js"
+        copy_file "README"
+      end
+
+      def copy_appropriate_package_json
+        if options.with_redux?
+          copy_file "client/redux_package.json", "client/package.json"
+        else
+          copy_file "client/package.json", "client/package.json"
+        end
+      end
+
+      def copy_appropriate_globals
+        if options.with_hello_world_example?
+          copy_file "hello_world_base/client/app/startup/clientGlobals.jsx", "client/app/startup/clientGlobals.jsx"
+        else
+          copy_file "client/app/startup/clientGlobals.jsx", "client/app/startup/clientGlobals.jsx"
+        end
+        return unless options.with_server_rendering?
+
+        if options.with_hello_world_example?
+          copy_file "hello_world_server_render/client/app/startup/serverGlobals.jsx",
+                    "client/app/startup/serverGlobals.jsx"
+        else
+          copy_file "client/app/startup/serverGlobals.jsx", "client/app/startup/serverGlobals.jsx"
+        end
+      end
+
+      def copy_appropriate_index_jade
+        if options.with_hello_world_example?
+          copy_file "hello_world_base/client/index.jade", "client/index.jade"
+        else
+          copy_file "client/index.jade", "client/index.jade"
+        end
+      end
+
+      def copy_appropriate_procfile
+        dev_procfile = options.with_server_rendering? ? "Procfile.dev.server" : "Procfile.dev.client"
```

Comment:
it's not really "Procfile.dev.server" but "Procfile.dev.with_server" or  "Procfile.dev.client_only"